nautobot/extras/templates/extras/role_retrieve.html

```diff
@@ -191,4 +191,20 @@
         </div>
     {% include 'inc/paginator.html' with paginator=controller_table.paginator page=controller_table.page %}
     {% endif %}
+    {% if module_table %}
```

Comment:
Should we similarly update the `location` detail view to include a count of colocated modules?